Index: vte-ng/src/vteconv.cc
===================================================================
--- vte-ng.orig/src/vteconv.cc
+++ vte-ng/src/vteconv.cc
@@ -771,7 +771,7 @@ int
 main (int argc,
       char *argv[])
 {
-        g_test_init (&argc, &argv, NULL);
+        g_test_init (&argc, &argv, (void *)0);
 
         g_test_add_func ("/vte/conv/utf8/strlen", test_utf8_strlen);
         g_test_add_func ("/vte/conv/utf8/validate", test_utf8_validate);
Index: vte-ng/src/vtetypes.cc
===================================================================
--- vte-ng.orig/src/vtetypes.cc
+++ vte-ng/src/vtetypes.cc
@@ -407,7 +407,7 @@ test_util_smart_fd(void)
 int
 main(int argc, char *argv[])
 {
-        g_test_init (&argc, &argv, NULL);
+        g_test_init (&argc, &argv, (void *)0);
 
         g_test_add_func("/vte/c++/grid/coords", test_grid_coords);
         g_test_add_func("/vte/c++/grid/span", test_grid_span);
